kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/hspa/HspaModem.java

```diff
@@ -39,48 +45,50 @@
 
 public class HspaModem implements HspaCellularModem {
 
-    private static final Logger s_logger = LoggerFactory.getLogger(HspaModem.class);
+    private static final Logger logger = LoggerFactory.getLogger(HspaModem.class);
 
-    protected int m_pdpContext = 1;
+    protected static final String MODEM_NOT_AVAILABLE_FOR_AT_CMDS_MSG = "Modem not available for AT commands";
 
-    protected static Object s_atLock = new Object();
+    // FIXME PDP context should not be hard coded
+    protected int pdpContext = 1;
 
-    protected String m_model;
-    protected String m_manufacturer;
-    protected String m_serialNumber;
-    protected String m_revisionId;
-    protected int m_rssi;
-    protected Boolean m_gpsSupported;
-    protected String m_imsi;
-    protected String m_iccid;
+    protected static Object atLock = new Object();
 
-    private ModemDevice m_device;
-    private final String m_platform;
-    private final ConnectionFactory m_connectionFactory;
-    private List<NetConfig> m_netConfigs;
+    protected String model;
+    protected String manufacturer;
+    protected String serialNumber;
+    protected String revisionId;
+    protected int rssi;
+    protected Boolean gpsSupported;
+    protected String imsi;
+    protected String iccid;
 
-    public HspaModem(ModemDevice device, String platform, ConnectionFactory connectionFactory) {
+    private ModemDevice device;
+    private final String platform;
+    private final ConnectionFactory connectionFactory;
+    private List<NetConfig> netConfigs;
 
-        this.m_device = device;
-        this.m_platform = platform;
-        this.m_connectionFactory = connectionFactory;
+    public HspaModem(ModemDevice device, String platform, ConnectionFactory connectionFactory) {
+        this.device = device;
+        this.platform = platform;
+        this.connectionFactory = connectionFactory;
     }
 
     @Override
     public void reset() throws KuraException {
-        s_logger.warn("Modem reset not supported");
+        logger.warn("Modem reset not supported");
     }
 
     @Override
     public String getModel() throws KuraException {
-        synchronized (s_atLock) {
-            if (this.m_model == null) {
-                s_logger.debug("sendCommand getModelNumber :: {}", HspaModemAtCommands.getModelNumber.getCommand());
-                byte[] reply = null;
+        synchronized (atLock) {
+            if (this.model == null) {
+                logger.debug("sendCommand getModelNumber :: {}", HspaModemAtCommands.getModelNumber.getCommand());
+                byte[] reply;
                 CommConnection commAtConnection = openSerialPort(getAtPort());
```

Comment:
It's so unfortunate that we cannot use try-with-resources with CommConnection, unless we wrap it in an AutoCloseable stream